examples/foo-missionmodel/src/main/java/gov/nasa/jpl/aerie/foomissionmodel/Mission.java

```diff
@@ -37,9 +38,10 @@ public final class Mission {
   public final Clock utcClock = new Clock(Instant.parse("2023-08-18T00:00:00.00Z"));
   private final Registrar cachedRegistrar; // Normally bad practice, only stored to demonstrate built/unbuilt check
 
-  public Mission(final Registrar registrar, final Configuration config) {
+  public Mission(final Registrar registrar, Instant planStart, final Configuration config) {
```

Comment:
For the sake of consistency, we can tack a `final` on here.

Reply:
Good catch! Thanks

Reply:
Done :)